web-registry/src/lib/queries/react-query/registry-server/graphql/getProjectByIdQuery/getProjectByIdQuery.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  ProjectByIdDocument,
+  ProjectByIdQuery,
+  ProjectByIdQueryVariables,
+} from 'generated/graphql';
+
+import { getProjectByIdKey } from './getProjectByIdQuery.constants';
+import {
+  ReactQueryProjectByIdProps,
+  ReactQueryProjectByIdResponse,
+} from './getProjectByIdQuery.types';
+
+export const getProjectByIdQuery = ({
+  id,
+  client,
+  ...params
+}: ReactQueryProjectByIdProps): ReactQueryProjectByIdResponse => ({
+  queryKey: getProjectByIdKey(id),
+  queryFn: async () => {
+    try {
+      const projectById = await client.query<
+        ProjectByIdQuery,
+        ProjectByIdQueryVariables
+      >({
+        query: ProjectByIdDocument,
+        variables: { id },
+        fetchPolicy: 'no-cache',
```

Comment:
using no-cache here since we're wrapping this with react-query and invalidating queries wouldn't work with apollo based caching enabled

Reply:
Make sense for using `no-cache` here but shouldn't we use the in-memory `graphqlClient` instead or is there a reason to use this one?

Reply:
we use the in-memory `graphqlClient` under the hood, which is passed as a parameter
I believe it's best to keep it as is to allow passing a more customized client in the future (thinking about keplr login)